### PR TITLE
feature: added jwt auth token functionality

### PR DESCRIPTION
**Pull request by rajeshj11**

### CODE REVIEW

**Review by Himasnhu-AT — APPROVED** [review]
Thanks @rajeshj11

**Comment by Himasnhu-AT** [comment]
Can you try resolving the merge conflicts

**Comment by rajeshj11** [comment]
resolving, will update you once it is done

**Review by Himasnhu-AT — APPROVED** [review]

File: src/main.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 import { AppModule } from './app.module';
 import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import * as cookieParser from 'cookie-parser';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
-  app.enableCors();
+  app.enableCors({
+    origin: ['http://localhost:3000'],
```
Comment (rajeshj11) [inline src/main.ts:10]: @Himasnhu-AT I think we need to add this in **https://web-client-ai-res.vercel.app/**. This change is breaking the site.

**Pull request merged**